### ISSUE TITLE
CMS error when clicking an asset

### ISSUE THREAD

**Issue by Teranode** [octobercms/october#4376]
- OctoberCMS Build: 6d283829c04098ddb7e0f4700c0078864831806e
- PHP Version: 7.1
- Database Engine: MariaDB 10.2
- Plugins Installed: Demo

### Description:
Upon clicking an asset you will get this error. For now i just removed the type-hinting references
```
"Type error: Argument 1 passed to Cms\Controllers\Index::canCommitTemplate() must be an instance of Cms\Classes\CmsObject, instance of Cms\Classes\Asset given, called in /home/genenxsq/public_html/modules/cms/controllers/Index.php on line 138" on line 486 of /home/genenxsq/public_html/modules/cms/controllers/Index.php
```

### Steps To Reproduce:
Go to CMS
go to assets
click on an asset

**Comment by willouch** [comment]
Same error php 7.3

**Comment by bennothommo** [comment]
Fix is now in place as of https://github.com/octobercms/october/commit/a777c44cb43a4a53882103dc9904cf18c67ba11b.

**Comment by Asinox** [comment]
Same here!

**Comment by LukeTowers** [comment]
@Asinox please use the latest dev branch until the next build is out:

> Fix is now in place as of a777c44.
